specs/_features/maxeb_increase/capella.py

```diff
@@ -3540,24 +3558,40 @@ def has_eth1_withdrawal_credential(validator: Validator) -> bool:
     return validator.withdrawal_credentials[:1] == ETH1_ADDRESS_WITHDRAWAL_PREFIX
 
 
+def has_compounding_withdrawal_credential(validator: Validator) -> bool:
+    """
+    Check if ``validator`` has an 0x02 prefixed "compounding" withdrawal credential.
+    """
+    return validator.withdrawal_credentials[:1] == COMPOUNDING_WITHDRAWAL_PREFIX
+
+
 def is_fully_withdrawable_validator(validator: Validator, balance: Gwei, epoch: Epoch) -> bool:
     """
     Check if ``validator`` is fully withdrawable.
     """
     return (
-        has_eth1_withdrawal_credential(validator)
+        (has_eth1_withdrawal_credential(validator) or has_compounding_withdrawal_credential(validator))
         and validator.withdrawable_epoch <= epoch
         and balance > 0
     )
 
+def get_validator_excess_balance(validator: Validator, balance: Gwei) -> Gwei:
+    """
+    Get excess balance for paritial withdrawals for ``validator``.
+    """
+    if has_compounding_withdrawal_credential(validator) and balance > MAX_EFFECTIVE_BALANCE:
+        return balance - MAX_EFFECTIVE_BALANCE
+    elif has_eth1_withdrawal_credential(validator) and balance > MIN_ACTIVATION_BALANCE:
+        return balance - MIN_ACTIVATION_BALANCE
+    return 0
 
 def is_partially_withdrawable_validator(validator: Validator, balance: Gwei) -> bool:
     """
     Check if ``validator`` is partially withdrawable.
     """
-    has_max_effective_balance = validator.effective_balance == MAX_EFFECTIVE_BALANCE
-    has_excess_balance = balance > MAX_EFFECTIVE_BALANCE
-    return has_eth1_withdrawal_credential(validator) and has_max_effective_balance and has_excess_balance
+    excess_balance = get_validator_excess_balance(validator, balance)
+    has_excess_balance = excess_balance > 0
+    return has_excess_balance
```

Comment:
consider collapsing into just one line `return get_validator_excess_balance(validator, balance) > 0`

Reply:
done!

Reply:
Not seeing it on my end :) Did you push?

<img width="940" alt="Screenshot 2023-05-29 at 13 32 45" src="https://github.com/michaelneuder/consensus-specs/assets/731743/5259a0f0-dc0a-4af0-a89b-543433de5784">